src/main/macros/jython/PyramidConversion.py

```diff
@@ -0,0 +1,58 @@
+# imports
+from loci.plugins import BF
+from loci.plugins.in import ImporterOptions
+from loci.formats import ImageReader
+from loci.formats import MetadataTools
+from loci.formats import FormatTools
+from loci.formats.out import OMETiffWriter
+from loci.common.image import IImageScaler
+from loci.common.image import SimpleImageScaler
+from ome.xml.model.primitives import PositiveInteger
+from ome.units import UNITS
+from java.lang import Math
+from ij import IJ
+
+# settings
+file = "/path/to/inputFile.tiff"
```

Comment:
file = IJ.getFilePath("Choose a Tiff File")
if file is None:
print("cancel")
sys.exit(2)

path = ('.').join(file.split('.')[:-1])
outFile = path + ".ome.tiff"